Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:3.10-slim-bullseye
+
+USER root
+
+# Setup env
+ENV LANG C.UTF-8
+ENV LC_ALL C.UTF-8
+
+RUN useradd -ms /bin/bash manticore
+
+USER manticore
+WORKDIR /home/manticore
+
+COPY Pipfile.lock /home/manticore
+COPY . /home/manticore
+
+RUN pip install pipenv
+RUN python -m pipenv install --system --deploy --ignore-pipfile
```

Comment:
What behaviour do we want here exactly? Are we trying to enforce the lock file or not for deployment?
https://pipenv.pypa.io/en/latest/advanced/#using-pipenv-for-deployments

Reply:
TBH I just found it as a reccomended way of installation in Docker image :)
`--system` to not create venv - just install globally in docker image.
`--deploy` to fail if `Pipfile.lock` is somehow out of date in comparison to `Pipfile`.
`--ingore-pipfile` to always use locked reqs, not reqs available from specified version boundaries from `Pipfile`.

Reply:
Ahh okay okay - thanks for the explanation! 🥰